lib/instances/Humanoid_spec.lua

```diff
@@ -0,0 +1,51 @@
+local Humanoid = import("./Humanoid")
+local Workspace = import("./Workspace")
+
+describe("instances.Humanoid", function()
+	it("should instantiate", function()
+		local instance = Humanoid:new()

```

Comment:
```suggestion
		local instance = Instance.new("Humanoid")
```

Part of this test's utility is making sure that Humanoid is actually creatable. You'll also need to add an import for `Instance` to this file.

Reply:
Just as I was about to say how cool GitHub suggested changes are, I see that you couldn't just write the import yourself with it :(

(Fixing it)

Reply:
Fixed.